CMakeLists.txt

```diff
@@ -36,6 +36,11 @@ ign_find_package(TINYXML2 REQUIRED PRIVATE PRETTY tinyxml2)
 ign_find_package(ignition-common3 REQUIRED PRIVATE)
 set(IGN_COMMON_MAJOR_VER ${ignition-common3_VERSION_MAJOR})
 
+#--------------------------------------
+# Find ignition-utils
+ign_find_package(ignition-utils1 REQUIRED COMPONENTS cli)
+set(IGN_UTILS_VER ${ignition-utils1_VERSION_MAJOR})
```

Comment:
We wouldn't want to add a new dependency to a stable branch. If this is needed to fix the tests, let's just disable them. I'm doing so in #255, so maybe that could be backported instead.

Reply:
no, it's not for the tests, it's using the `cli` component for the standalone executables:

* https://github.com/gazebosim/gz-launch/pull/256/files#diff-bdcc4dee417bf00f171ecda26e5bcb5670ab4a129dbd2ee3b9b21fe15918b895R22

I'll see if we can backport some of the cmake changes without switching to a standalone executable

Reply:
@scpeters were you able to take a look to that? let us know if we can help you on that one